src/storage/test/TestUtils.h

```diff
@@ -117,13 +115,40 @@ class TestUtils {
         return vertices;
     }
 
+    static std::vector<cpp2::Edge>
+    setupEdges(PartitionID part, VertexID srcFrom,
+               VertexID srcTo, std::string tmp = "%d_%ld") {
```

Comment:
How about rename `tmp` to `fmt` or some other meaningful name?

Reply:
yep ,`fmt` is a good choice.